.env.example

```diff
@@ -1,13 +1,12 @@
-SECRET_SALT=
-RABBITMQ_HOST=
-RABBITMQ_PASSWORD=
-RABBITMQ_PORT=
-RABBITMQ_USER=
-SECRET_KEY_BASE=
-AUTH_USER=
-AUTH_PASS=
-AUTH_REALM=Admin Area
-METAPHYSICS_URL=https://metaphysics-staging.artsy.net
-EXCHANGE_URL=https://exchange-staging.artsy.net
-SLACK_API_TOKEN=REPLACE
-SLACK_SLASH_COMMAND_TOKEN=REPLACE
```

Comment:
^^ some of these moved to `.env.aprd` in s3. The new `bin/setup` script pulls the file down.